overlord/devicestate/export_test.go

```diff
@@ -83,6 +83,14 @@ func SetLastBecomeOperationalAttempt(m *DeviceManager, t time.Time) {
 	m.lastBecomeOperationalAttempt = t
 }
 
+func SetRunMode(m *DeviceManager, mode string) {
+	m.runMode = mode
+}
+
+func RunMode(m *DeviceManager) string {
+	return m.runMode
```

Comment:
this is fine for now, at some point we will probably need to have DeviceManager.Mode